src/swupd.h

```diff
@@ -164,6 +164,8 @@ extern int get_absolute_latest_version(void);
 extern enum swupd_code read_versions(int *current_version, int *server_version, char *path_prefix);
 extern int get_current_version(char *path_prefix);
 extern bool get_distribution_string(char *path_prefix, char *dist);
+extern void get_current_format(char **current_format);
+extern int get_latest_format(int server_version);
```

Comment:
Rename this to get_server_format(). You only use this one for the latest, but this function can be used to get the format number of any server version.

Reply:
Done